### PR TITLE
Sprint 1 review+retrospective

### PR DESCRIPTION
**Pull request by escapewindow**

### CODE REVIEW

File: sprints/20201110-apple-silicon-fenix-shipit/sprint-review.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Or, "There's no perfect time to start sprints".
+
+While members of the Releng team had participated in scrum at other companies, a number of us were new to agile, and none of us had Scrum Master experience. But we knew we needed to change how we worked with our reduced team size and combined team structure.
+
+A number of team members had PTO booked, meaning there was no perfect two-week span for our first sprint before US Thanksgiving week. We decided that starting the sprints on an imperfect week was preferable over waiting for some mythical perfect time.
+
+Aki had the most background from the [Taskcluster team's sprints](https://github.com/taskcluster/scrum/) and planned to help keep things on track in week 2, after coming back from PTO. However, life happened, and he was out both weeks, on bereavement leave.
+
+We tried two parallel sprints, one per epic, and some non-sprinting team members to handle interrupts and releaseduty. We largely didn't use JIRA to track things. There was no Product Owner on the Apple Silicon sprint. Mihai was understandably unable to handle both his new management role and the Scrum Master role, so we were Scrum-Master-less as well.
```
Comment (MihaiTabara) [inline sprints/20201110-apple-silicon-fenix-shipit/sprint-review.md:11]: Oups, mea culpa again for this. O_O

**Review by MihaiTabara — APPROVED** [review]
Thanks a lot for importing this here! Do you think we should clean retroactively the content from the https://docs.google.com/document/d/15DS0CHEKQ0OvgKpJ5ygOH3CTcYxZegz6SP-gyQMFRJ8/edit#heading=h.kcc4eaanbp0g doc so that we can keep that just as taking notes during retrospectives?

**Comment by escapewindow** [comment]
> Thanks a lot for importing this here! Do you think we should clean retroactively the content from the https://docs.google.com/document/d/15DS0CHEKQ0OvgKpJ5ygOH3CTcYxZegz6SP-gyQMFRJ8/edit#heading=h.kcc4eaanbp0g doc so that we can keep that just as taking notes during retrospectives?

Could do. The doc is getting a bit long. We could also start a new doc per sprint, and link to the previous/next (or have a single location to find all the docs)?

File: sprints/20201110-apple-silicon-fenix-shipit/sprint-review.md (hunk 1 above)
Comment (escapewindow) [reply sprints/20201110-apple-silicon-fenix-shipit/sprint-review.md:11]: No worries. Trying to just describe what happened; no blame here.

**Comment by escapewindow** [comment]
Merging without sprints 2 and 3, so others can work on those if wanted.

**Comment by MihaiTabara** [comment]
> > Thanks a lot for importing this here! Do you think we should clean retroactively the content from the https://docs.google.com/document/d/15DS0CHEKQ0OvgKpJ5ygOH3CTcYxZegz6SP-gyQMFRJ8/edit#heading=h.kcc4eaanbp0g doc so that we can keep that just as taking notes during retrospectives?
> 
> Could do. The doc is getting a bit long. We could also start a new doc per sprint, and link to the previous/next (or have a single location to find all the docs)?

Good poing. Done. Added link to the directory at the top of the RelEng Sprint iterations docs.

**Comment by MihaiTabara** [comment]
Added sprint 2 info and skeleton for sprint 4 in https://github.com/mozilla-releng/scrum/pull/6

**Pull request merged**